airflow/contrib/hooks/emr_hook.py

```diff
@@ -63,6 +63,8 @@ def create_job_flow(self, job_flow_overrides):
             VisibleToAllUsers=config.get('VisibleToAllUsers'),
             JobFlowRole=config.get('JobFlowRole'),
             ServiceRole=config.get('ServiceRole'),
+            AutoScalingRole=config.get('AutoScalingRole'),
+            ScaleDownBehavior=config.get('ScaleDownBehavior'),            
             Tags=config.get('Tags')
```

Comment:
Hmmm, I wonder if it might be an idea to do `self.get_conn().run_job_flow(**config)` - that we don't have to stay up-to-date with the AWS API.

For that to work we'd need three `config.setdefault()` calls for the three list-types. I also don't know if all the kwargs are optional or not (but I suspect they are).